reader.go

```diff
@@ -1578,3 +1578,14 @@ func extractTopics(members []GroupMember) []string {
 
 	return topics
 }
+
+func describeOffset(offset int64) string {
+	switch offset {
+	case FirstOffset:
+		return "(first offset)"
+	case LastOffset:
+		return "(last offset)"
```

Comment:
Maybe the resulting log messages would read more naturally if we removed the surrounding parenthesis?
```suggestion
		return "first offset"
	case LastOffset:
		return "last offset"
```

Reply:
Done.